.github/workflows/main.yml

```diff
@@ -0,0 +1,38 @@
+name: 'Push to Github Container Registry'
+
+on:
+    pull_request:
+        types: closed
```

Comment:
Is there a types: merged? I don't think it's a huge deal, but this would trigger on non-merged closed PRs as well.

Reply:
That's a good point. Surprisingly, Github Actions doesn't offer a `types: merged`. It's still possible, but a bit more work. I'll open an issue for that separately.